Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainViewModel.swift

```diff
@@ -160,40 +166,47 @@ final class PlaygroundMainViewModel: ObservableObject {
         ) { [weak self] setupPlaygroundResponse in
             if let setupPlaygroundResponse = setupPlaygroundResponse {
                 PresentFinancialConnectionsSheet(
-                    setupPlaygroundResponseJSON: setupPlaygroundResponse
-                ) { result in
-                    switch result {
-                    case .completed(let session):
-                        let accounts = session.accounts.data.filter { $0.last4 != nil }
-                        let accountInfos = accounts.map { "\($0.institutionName) ....\($0.last4!)" }
-                        let sessionInfo =
+                    setupPlaygroundResponseJSON: setupPlaygroundResponse,
+                    onEvent: { event in
+                        if self?.showLiveEvents == true {
```

Comment:
note that this code is in the "Examples" directory which by default isn't very polished (as long as it works OK its fine)